packages/theming/README.md

```diff
@@ -14,43 +14,33 @@ npm install react react-dom prop-types styled-components @zendeskgarden/react-th
 
```

Comment:
Should we remove the `retrieveComponentStyles` ID table in the theming docs? Not sure how useful that will be now that the CSS-in-JS usages won't be found. This would also require removing the logic from `styleguide.base.config.js`